packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -10,21 +10,60 @@
 @import '~carbon-components/scss/globals/scss/layout';
 @import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
 @import '~carbon-components/scss/globals/scss/css--reset';
+@import '~carbon-components/scss/components/ui-shell/ui-shell';
+@import '~carbon-components/scss/components/ui-shell/side-nav';
 
-/// Masthead styles
 /// @access private
-/// @group masthead
+/// @group dotcom ui-shell
+
+$search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
+$search-transition-timing: 95ms;
+
 @mixin masthead {
```

Comment:
Can we not use theming for this?

```scss
@import "~@carbon/themes/scss/themes";

@include carbon--theme($carbon--theme--white) {
  @import '~carbon-components/scss/components/ui-shell/ui-shell';
}

Reply:
Looking at it more closely it looks like they aren't yet using the theming variables, but it looks like they are providing component level tokens. Maybe we could look at reassigning them, and theming it that way before overriding what they have in here? Maybe you already looked into this though?

https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/components/ui-shell/_theme.scss